include/revng/PipelineC/Prototypes.h

```diff
@@ -316,8 +333,6 @@ void rp_target_destroy(rp_target *target);
 rp_kind *rp_target_get_kind(rp_target *target);
 
 /**
- * Serializes target into a string, target cannot be NULL.
- *
```

Comment:
Why?

Reply:
erased the entire line by accident, meant to only erase about the NULL thing